### ISSUE TITLE
G.Vector(G.993.5)DSL AX82U

### ISSUE THREAD

**Issue by trebleta** [gnuton/asuswrt-merlin.ng#206]
**Is your feature request related to a problem? Please describe.**
In DSL setting vector state is unconfiged

**Describe the solution you'd like**
Would be cool to have the option to enable/ disable G.Vector also to have the option of non standard g vector.

**Describe alternatives you've considered**
None

**Additional context**
![image](https://user-images.githubusercontent.com/43239579/176904467-6575c8f1-08e4-4389-9569-67912097195e.png)

**Comment by Yakumounr** [comment]
"Vectoring State Unconfigured" is normal for a line that doesn't have vectoring enabled by the ISP at the cabinet. BT/OpenReach in the UK for example has been rolling it out very slowly, there was news coverage of BT extending vectoring trials way back in 2014, but also customer posts questioning when they would be deploying it in 2019. 

BT paused it's G.Fast roll out in 2020 in favour of FTTP, perhaps they paused vectoring also but without news coverage, or as DreyTek mention it is not expected to be of much benefit on long lines so it may just be few customers have it enabled, either way it doesn't seem to be discussed online much in the UK any more.

**Comment by trebleta** [comment]
Yes from what I can tell it's been stoped, why wanted to know if can get added so would be disabled, also there is a test you can do see if you do get v state. At ssh prompt type xdslctl info ~~vectoring
If you see in the info vectoring state 5 its disabled, if you see a 1 its enabled
